frost-core/src/frost/keys.rs

```diff
@@ -754,39 +756,20 @@ pub fn reconstruct<C: Ciphersuite>(
 
     let mut secret = <<C::Group as Group>::Field>::zero();
 
-    // Compute the Lagrange coefficients
-    for (i_idx, i, secret_share) in secret_shares
+    let xset: BTreeSet<_> = secret_shares
```

Comment:
aha, any problem using similar variable names out here and inside `compute_lagrange_coefficients()`?

Reply:
Good point. I went for `identifiers` in a3558ea which seems more semantic, what do you think?